frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/LauncherConfiguration.java

```diff
@@ -95,6 +95,17 @@
   private Integer amSetupContainerRequestMinRetryIntervalSec = 30;
   private Integer amSetupContainerRequestMaxRetryIntervalSec = 150;
 
+  // Small ports usually reserved for system usage,  the minimum port a job can use.
+  private Integer amContainerMinPort = 2000;
+  private Integer amSearchNodeBufferFactor = 2;
```

Comment:
>amSearchNodeBufferFactor [](start = 18, length = 24)

Comment what is amSearchNodeBufferFactor

Reply:
Why name it a buffer ?

---
In reply to: [174682595](https://github.com/Microsoft/pai/pull/318#discussion_r174682595) [](ancestors = 174682595)